docs/components/servo/gpio.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Configure a gpio Servo"
+linkTitle: "gpio"
+weight: 90
+type: "docs"
+description: "Configure a gpio servo."
+tags: ["servo", "components"]
+icon: "img/components/servo.png"
+# SME: Rand
+---
+
+Configure a `gpio` servo to integrate a hobby servo controlled by general-purpose input/output (GPIO) pins on a non-`pi` model [board](/components/board) into your robot:
+
+{{< tabs name="Configure a gpio Servo" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your servo, select the type `servo`, and select the `gpio` model.
+
+Click **Create component**:
+
+![An example configuration for a gpio servo in the Viam app Config Builder.](../img/gpio-servo-ui-config.png)
+
+Edit and fill in the `"attributes"` JSON to align with your `board` `name` and GPIO wiring.
```

Comment:
```suggestion
Edit and fill in the attributes with your board `name` and GPIO wiring.
```
I don't think attributes needs to be stylized since it's a UI element "Attributes" and then it's also a JSON struct, and we want to align with fancy mode so if anything it could be **Attributes**, or just leave it conceptual (and therefore unstylized).

Or actually, to match the alignment PR:
```suggestion
Edit and fill in the attributes as applicable.
```